ingest/src/main/java/com/microsoft/azure/kusto/ingest/QueuedIngestClientImpl.java

```diff
@@ -57,6 +59,8 @@ public class QueuedIngestClientImpl extends IngestClientBase implements QueuedIn
         this.resourceManager = new ResourceManager(client, httpClient);
         this.azureStorageClient = new AzureStorageClient();
         this.connectionDataSource = csbWithEndpoint.getClusterUrl();
+        this.applicationForTracing = csb.getApplicationNameForTracing();
```

Comment:
Create a client details object and use that - these properties are not formatted/have the wrong defaults

Reply:
When looking at the client details object current usage - the appendedClientVersionForTracing is either null, or csb.getApplicationNameForTracing() as done here. so not sure why this is wrong

Reply:
```
 constructor(public applicationNameForTracing: string | null, public userNameForTracing: string | null) {
        if (this.applicationNameForTracing === null) {
            this.applicationNameForTracing = ClientDetails.defaultApplication();
        }
        if (this.userNameForTracing === null) {
            this.userNameForTracing = ClientDetails.defaultUser();
        }
        this.versionForTracing = ClientDetails.defaultVersion();
    }
````

Reply:
![image](https://github.com/Azure/azure-kusto-java/assets/150662318/e954fc88-e2a0-4fab-9d87-51fd104e470d)
this is the usage so far as well in the client details too though - so using clientdetails would give you the exact same thing

Reply:
The getter in ClientDetails returns the default if it's null - that's the piece of code I showed you.
I'm not sure what do you mean

Reply:
ohhh in applilcation tracing, all the time i looked at the second one, which is indded the same. ok, changing